src/main/java/io/aiven/kafka/connect/opensearch/OpensearchSinkConnectorConfig.java

```diff
@@ -164,11 +164,19 @@ public class OpensearchSinkConnectorConfig extends AbstractConfig {
             IndexWriteMethod.UPSERT.name().toLowerCase(Locale.ROOT));
 
     public static final String DATA_STREAM_ENABLED = "data.stream.enabled";
+    public static final String DATA_STREAM_CREATE_INDEX_TEMPLATE = "data.streams.create.index.template";
```

Comment:
Why do we need 2 properties? Could we use one, fe `data.streams.index.template` and if no set - implicitly assume we have to create one?

Reply:
Agree, we can just use one, but it would be more clear for the user, that we are creating a template with the provided template name, if the other config is set to true. But if you think it's redundant, I shall update code to have only one config.

Reply:
@reta updated PR description and pr too. 

If data.streams.existing.index.template.name is provided, and if that index template does not exist, a template will be created with that name, else no template is created.

If data.streams.existing.index.template.name is not provided, a default template is always created